src/handlers/xp.js

```diff
@@ -5,6 +5,14 @@ const { environment } = require('../environment')
 const MessageEmbed = require('discord.js').MessageEmbed
 const NodeAddress = require('../sourcecred-utils')
 
+const filterMultipleDiscordAccounts = (discordAccount, targetUserDiscordID) => {
+  const discordId = NodeAddress.toParts(discordAccount.address)[4]
+  if (discordId === targetUserDiscordID) {
+    return discordId
+  }
+  return undefined
```

Comment:
NIT: I'd probably return null here, unless there is a specific reasoning for returning undefined

Reply:
Answered below 👍